build_sdk.sh

```diff
@@ -23,7 +22,7 @@ function make_python_sdk() {
 if [ $# -ne 1 ]; then
     echo "Usage: $0 generating-language"
     echo "Supported languages: cpp, javascript, python"
-    exit -1
+    exit 1
```

Comment:
Just curious, what's the reason behind this change?

Reply:
> Note in particular that `exit -1` is equivalent to `exit 255`, but that `exit 1` is the more canonical way of expressing the first possible error code.
- https://www.shellcheck.net/wiki/SC2242
- https://tldp.org/LDP/abs/html/exitcodes.html